mxcubeweb/app.py

```diff
@@ -202,6 +203,22 @@ def init_signal_handlers():
         MXCUBEApplication.beamline.init_signals()
         MXCUBEApplication.harvester.init_signals()
 
+    @staticmethod
+    def _get_graylog_handler(config, log_level):
+        server_cfg = getattr(config, "server", None)
+        graylog_host = getattr(server_cfg, "GRAYLOG_HOST", None)
+        graylog_port = getattr(server_cfg, "GRAYLOG_PORT", None)
+        if graylog_host and graylog_port:
+            try:
+                handler = graypy.GELFUDPHandler(graylog_host, graylog_port)
+                handler.setLevel(log_level)
+            except Exception as ex:
+                msg = "Graylog handler could not be initialized: " + str(ex)
+                logging.getLogger("HWR").info(msg)
+            else:
+                return handler
```

Comment:
```suggestion
            try:
                handler = graypy.GELFUDPHandler(graylog_host, graylog_port)
            except Exception as ex:
                msg = "Graylog handler could not be initialized: " + str(ex)
                logging.getLogger("HWR").info(msg)
            else:
                handler.setLevel(log_level)
                return handler
```

It's always a good idea to have as short `try`-blocks as possible ( ideally one expression long :smile_cat: )